model/README.md

```diff
@@ -1,9 +1,14 @@
 ## Pre-requsites
-1. python 3.6.12 or higher
+1. python 3.9.6 or higher
 
 ## Installing python libraries
 `pip install -r requirements.txt`
 
+## Running on Mac without Docker
+1. install git lfs - `brew install git-lfs && git lfs install`
+2. clone pegasus model - `cd pegasus_paraphrase && git lfs pull`
+3. run app - `uvicorn main:app --reload`
+
 ## Running the project
 ```
 # build docker container
```

Comment:
watch out for the inconsistencies below in the readme:
docs should be found on `localhost:3000/docs` based on the port mapping you show in the Running the project section